config/Readme.md

```diff
@@ -0,0 +1,40 @@
+# IMAGE Config
+
+This Folder (Config) should contain the .env files required for IMAGE server preprocessors to run precisely.
+
+## Environment Files
+
+### maps.env
+This env file is required by the following preprocessors:
+* [autor preprocessor](https://github.com/Shared-Reality-Lab/IMAGE-server/tree/main/preprocessors/autour)
+* [openstreetmap preprocessor](https://github.com/Shared-Reality-Lab/IMAGE-server/tree/main/preprocessors/openstreetmap)
+
+Following is the sample format of maps.env file:
+```
+GOOGLE_PLACES_KEY = [INSERT KEY STRING]
+```
+
+### apis-and-selection.env
+This env file is required by the following preprocessors:
+* [ocr-clouds-preprocessor](https://github.com/Shared-Reality-Lab/IMAGE-server/tree/main/preprocessors/ocr)
+
+The desired cloud service is determined by an environment variable called CLOUD_SERVICE, it is a string and the possible values are:
```

Comment:
Again: explain where these API keys need to come from, and especially the difference between `AZURE_OCR` and `AZURE_READ`.